src/app/pages/website/homepage/homepage.component.html

```diff
@@ -1,4 +1,4 @@
-<littil-content-container outerClasses="mb-3">
+<littil-content-container outerClasses="mb-3" innerClasses="prose">
```

Comment:
Als de `prose` class op alle littil-content-containers nodig zijn, kunnen we hem dan niet default toevoegen in het component zelf? Hetzelfde geldt eigenlijk ook voor de `mb-3` class

Reply:
Daar valt wat voor te zeggen. Bij sommige content wil je prose niet toepassen omdat dit bijeffecten , maar dat zie je vanzelf. Je kunt altijd no-prose gebruiken om het uit te schakelen. Ik zal het in de volgende commits doorvoeren